### ISSUE TITLE
[UX] combine different kinds of views link fields

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#3457]
Recommended by @BWPanda in https://github.com/backdrop/backdrop-issues/issues/3419

> As an extra feature request, and slightly different from Drupal 8, what about also merging all the link fields together? E.g. remove 'Content: Delete link' and 'Content: Edit link' in favour of a more powerful 'Content: Link'.

Here's how it could look:

![screenshot_2018-12-14 administer content content panda s backdrop sandbox 1](https://user-images.githubusercontent.com/2385329/49997384-1bb9ad80-ff9a-11e8-803b-167b5cf3deb5.png)
(I couldn't screenshot the select list open, but it'd have 'View', 'Edit', & 'Delete' as options.)

**Comment by BWPanda** [comment]
Here's the commit I made to achieve this: https://github.com/backdrop/backdrop/pull/2425/commits/6a51ef7c26b92462184354683db55d9ea1b1cbda

And here's that commit as a patch file in case that code goes away somehow in the future: [merge_link_fields.patch.txt](https://github.com/backdrop/backdrop-issues/files/5044556/merge_link_fields.patch.txt)

**Comment by olafgrabienski** [comment]
> remove 'Content: Delete link' and 'Content: Edit link' in favour of a more powerful 'Content: Link'

Hm, when I'd look for a delete link in Views, I wouldn't guess it's 'behind' the Content link.

**Comment by yorkshire-pudding** [comment]
> > remove 'Content: Delete link' and 'Content: Edit link' in favour of a more powerful 'Content: Link'
> 
> Hm, when I'd look for a delete link in Views, I wouldn't guess it's 'behind' the Content link.

I agree. Another point is when you add multiple links as you're combining, but then the view interface shows them all as "Content: Link" unless you edit the admin label.
